### ISSUE TITLE
Asking about Guidance class.

### ISSUE THREAD

**Issue by nakeeun** [feihuzhang/GANet#54]
Thank you for your amazing work. GANet is really impressive with two cost aggregation layer based on SGM, CVF. 
I have question about your GANet code. When i research your paper and your code, i think code and paper in Eq.(4), Eq.(7) not matched, especially weights. 
First of all, in module.GANet.py,  g0, g1, g2, g3 are weights in SgaFunction. 
However, in sga_cuda_forward from src.GANet_cuda.cu, those(g0,g1,g2,g3) variable name are changed with guidance_down, up, right, left, respectively. Even these variables go into each sga direction function(such as sga_left_foward, sga_right_backward...) 
Example is as below.
![example](https://user-images.githubusercontent.com/37968684/65746909-01bd1600-e13b-11e9-8a88-3a20357c12a7.png)
Do i miss something??

**Comment by feihuzhang** [comment]
Yes, in the implementation g0 ... g1 means the guidance weights for aggregation in different directions.
For examples, g0 means aggregation down which has a set of weight g0={w0, w1... w4}.
